### DISCUSSION TITLE
createBreakpoints not available/installable - Host key verification failed.

### DISCUSSION THREAD

**Discussion by ivanjeremic** [chakra-ui/chakra-ui#2224] [Q&A] [answered]
`import { createBreakpoints } from "chakra-ui/theme-tools"`

I',m not able to import or install it directly from npm it says Host key verification failed. I would like to set up custom breakpoints like in the V1 documentation. [https://next.chakra-ui.com/docs/migration#7-update-theme-breakpoints](https://next.chakra-ui.com/docs/migration#7-update-theme-breakpoints)

```
Output:
Host key verification failed.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
```

**Comment by ljosberinn** [comment, marked as answer]
You have a typo there, `@chakra-ui/theme-tools`, not `chakra-ui`. Since that repo doesn't exist, npm/yarn can't read it.

**Reply by ivanjeremic** [reply]
Then the Docs have a typo, look at the imports here https://next.chakra-ui.com/docs/migration#7-update-theme-breakpoints

**Reply by ljosberinn** [reply]
ah yes, this has been [resolved in this commit](https://github.com/chakra-ui/chakra-ui/commit/2767f565507b156b5d29689115ad8e8fd6d7338b)! sorry for the inconvenience
